examples/route_guide/config/config.exs

```diff
@@ -1,3 +1,3 @@
 import Config
 
-# config :grpc, start_server: true
+config :grpc, start_server: true
```

Comment:
This is something I wish to refactor. IMO We should move to a "add X to your supervision tree" instead of this global config

Reply:
You means like:

```
children = [
  {GRPC.Server.Supervisor, {Endpoint, 50000}, start_server: true}
]
```

Reply:
Exactly! I've opened #246 for this

Reply:
This would enable multiple servers to run from the same application in different ports

Reply:
Great! I will change the example to use the new way after #246 get merge into master. 🙇‍♂️